src/main/java/com/artipie/rpm/meta/XmlOthers.java

```diff
@@ -168,7 +173,12 @@ public Package changelog()
          */
         public Package changelog(final String author, final int epoch, final String content)
             throws XMLStreamException {
-            return this.changelog();
+            this.xml.writeStartElement("changelog");
+            this.xml.writeAttribute("epoch", String.valueOf(epoch));
```

Comment:
@paulodamaso why this attribute is called `epoch`? Did you find this somewhere in docs? As I can see from the examples in this repo and test bundles (you can find download links in `com.artipie.rpm.files.TestBundle`) the attribute is called `date`.

Reply:
@paulodamaso ping

Reply:
@olenagerasimova I've used the same pattern following `version(final int eopch, final String ver, final String rel)` method

Reply:
@g4s8 I think we should rename it to `date` to correspond all the examples (but I could not find direct indication in docs)... @g4s8 wdyt?

Reply:
@paulodamaso @olenagerasimova  I don't really know what should be the name, it was the task to investigate it and use correct name here. As I can see in Centos7 repo metadata files, they have two attributes: `author` and `date`, and xml text for changelog inside element http://mirror.centos.org/centos/7/os/x86_64/repodata/ see:
```xml
<package pkgid="8add14ea2d064c09802c48d3e78e1ef2fcbce35222fa55d6eb3e34425fb69613" name="389-ds-base" arch="x86_64">
  <version epoch="0" ver="1.3.10.1" rel="5.el7"/>
  <changelog author="Mark Reynolds &lt;mreynolds@redhat.com&gt; - 1.3.9.1-6" date="1557489600">- Bump version to 1.3.9.1-6
- Resolves: Bug 1705125 - ipa-replica-install with 389-ds-base-1.3.9.1-5.el7
- Resolves: Bug 1643772 - ds-replcheck should validate suffix exists and it's replicated
- Resolves: Bug 1647133 - Log warn instead of ERR when aci target does not exist.
- Resolves: Bug 1639192 - Request to add passwordSendExpiringTime in password policy objectclass</changelog>
```